CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - ERC1155 component and preset (#896)
-- Mixin implementations in components (#863)
```

Comment:
We need to move this now to the Unreleased section.

Reply:
I created a separate entry specifically for ERC115 and ERC1155Receiver in Unreleased